pages/Getting Started/Installation.md

```diff
@@ -174,14 +174,17 @@ Hyprland and related are in the default repository:
 
 {{% details title="Ubuntu*" closed="true" %}}
 
-Hyprland made it into the universe repos and can be installed with 
+Hyprland made it into the Ubuntu 24.10 Oracular Oriole universe repo and can be installed with 
 ```bash
 sudo add-apt-repository universe && sudo apt-get update && sudo apt-get install -y hyprland
 ```
-Build dependencies:
+> [!NOTE]
+> Above is for Ubuntu 24.10 (Unreleased) version
+
+For installing Hyprland from Source, install first the dependencies below:
```

Comment:
This only works in GitHub markdown. Here, we use
```md
{{< callout type=info >}}

text here

{{< /callout >}}
```

Reply:
Oh thanks now I know... I have resolved it.